### ISSUE TITLE
Custom Report Custom Layout: Data at the root level is invalid. Line 1, position 1 error

### ISSUE THREAD

**Issue by reiniroosh** [Microsoft/AL#2632]
There's some kind of issue with (probably) txt2al causing some problems with rdl files.:
The issue was closed, but it's still happening, as is noted in the thread, but I can't reopen it..
Message: <ii>Data at the root level is invalid. Line 1,position 1.</ii>
#1513

Actually, the custom report compiles, deploys, and runs fine, so it may be a slightly different issue.

However if I go to Report Layouts and try to change it to "Custom Report" NAV crashes.
The server "DynamicsNAV110" was unable to process the request. The application will close.

Or, from the Custom Report Layouts page, if I click New and select my report, it crashes as well. (Not if I choose the checkbox for the Word layout.)

AL 15.18771
NAV 11.0.22292

**Comment by StanislawStempin** [comment]
You're using an unsupported combination of AL and NAV, please see https://github.com/Microsoft/AL/wiki/Frequently-Asked-Questions#which-version-of-the-al-language-extension-should-i-use

**Comment by reiniroosh** [comment]
I tried it with CU5, same result.
AL 0.12.17091
NAV 11.0.22292.0

![image](https://user-images.githubusercontent.com/7827702/41728887-bfb893c2-752c-11e8-9890-8a4e866f028d.png)

**Comment by StanislawStempin** [comment]
If you can reproduce the issue on the latest developer preview builds (AL 0.17, BC 13.0) we will investigate it further. Otherwise please contact regular support channels.

**Comment by reiniroosh** [comment]
I haven't had a chance to try that yet.

Anyways. I ran into a bug in your product, and reported it as a courtesy, and as a knowledge base in case it happens to someone else.

I just want to clarify that I will leave it up to you and your team how to proceed.

I am just reporting a bug I ran in to on your product as a courtesy, and to create a knowledge base in case someone else runs into the issue as well.
